app/src/main/kotlin/com/simplemobiletools/keyboard/views/MyKeyboardView.kt

```diff
@@ -932,7 +938,15 @@ class MyKeyboardView @JvmOverloads constructor(context: Context, attrs: Attribut
         }
 
         val popupKey = mKeys[mCurrentKey]
-        val result = onLongPress(popupKey, me)
+        val result = if (popupKey.code == KEYCODE_EMOJI) {
+            ChangeLanguagePopup(this, onSelect = {
```

Comment:
this is messy, move this away from the result declaration

Reply:
Please check. @tibbi